crawler/plugins/systems/gpu_host_crawler.py

```diff
@@ -66,7 +67,12 @@ def _get_containerid_from_pid(self, pid):
             cont_pid = int(state['Pid'])
             cont_child_pids = self._get_children_pids(cont_pid)
             if pid in cont_child_pids:
-                return inspect['Name']
+                labels = inspect['Config']['Labels']
+                pod_ns = labels.get('io.kubernetes.pod.namespace', 'NA')
+                pod_name = labels.get('io.kubernetes.pod.name', 'NA')
+                training_id = labels.get('training_id', pod_name)
+                name = "{}.{}".format(pod_ns, training_id)
+                return name
```

Comment:
this become super k8/dlaas dependent, can you add a generic fall-back case?

Reply:
sure, for pod_name, i can add container name as fallback for non-k8s deployment.
for k8s-namespace, there's no fallback, so it will be marked 'NA'. It could be host-name, but since host-ip is already covered in the metric path, i don;t want to add redundant field.

Reply:
Thanks!